### DISCUSSION TITLE
ARC for project management

### DISCUSSION THREAD

**Discussion by Brilator** [nfdi4plants/ARC-specfication#15] [Comments and Suggestions]
Hi everyone, 

Dominik here, CEPLAS, HHU Düsseldorf. 

What I envision for ARC in the future, is that it captures virtually anything from starting a project idea (pre-data) to publishing it and beyond (data re-use). 

Typical feedback I get from the user community 
1. Can you recommend a project management platform (something like Atlassian Confluence / wiki, GitHub/GitLab, cloud services and or electronic lab notebooks) to initiate a collaboration (with internal / external partners) and structure my (lab) projects?
2. How does the ARC link to my ELN? 
3. Why would I bother about something like the ARC as long as I don't have my data ready (e.g. for submission to some repository)? 

Basically I want my answer to always be: **ARC**. 
- Start an ARC at the moment you start discussing an idea
- Document the idea e.g. in isa.investigation.xlsx
  - including contributors
  - experiment design
  - everything readily linked with ontologies
- ARC through elixir-AAI allows you to share with whomever (not just internal partners) 
- Share it and grow from there
  - run assays
  - produce data
  - re-iterate the project
- If you do it like this you'll benefit right away from using the ARC (not having to spend the time once you want to publish) 

I know, the points are basically the same old story from any talk about FAIRification. And although technically it's all there, at the moment some of the info is unstructured (i.e. "additional payload"), especially considering the standard not-git-affine lab biologist, who might even wish for visible snapshots of the iteration happening in isa.investigation.xlsx as the project develops. 
Most of our colleagues are very centered around their current habits and environment (e.g. tools like ELNs) and hard to transition to something new (especially if a well established functionality is missing). Let's give them project management and integration into daily lab routines. 

Surely nothing for now, but maybe for future versions.

**Comment by Brilator** [comment]
This contradicts l. 267 
> The ARC MUST NOT be *empty*: it MUST contain minimally a single assay or a single workflow.

I would suggest to change to: 
The ARC MUST NOT be *empty*: it MUST contain minimally an isa.investigation.xlsx.

**Reply by chgarth** [reply]
Agree: ARCs should be shareable if they only contain the investigation file. 

Spec updated (d2f7071) to read:

> A *publishable* ARCs MUST NOT be empty; it MUST contain minimally a single assay or a single workflow.
